filebeat/module/redis/log/manifest.yml

```diff
@@ -12,3 +12,7 @@ var:
 
 ingest_pipeline: ingest/pipeline.json
 prospector: config/log.yml
+
+pipeline_script:
```

Comment:
I wonder if we should rename this to `scripts` instead of `pipeline_script(s)` as in ES these are just scripts and not related to a pipeline as far as I know?

Reply:
AFAIK no. I added `pipeline_` prefix so it is unambiguous that is a pipeline setting, because manifest includes other types of settings, too. If it's not needed I am happy to remove the prefix.

Reply:
I think we can remove it. @tsg please object if not the case.

Reply:
I agree we can do without the prefix.